schematic/models/validate_manifest.py

```diff
@@ -62,7 +62,7 @@ def get_multiple_types_error(
         return ["NA", error_col, error_message, error_val]
 
     def validate_manifest_rules(
-        self, manifest: pd.core.frame.DataFrame, sg: SchemaGenerator, restrict_rules: bool, project_scope: List, access_token: str,
+        self, manifest: pd.core.frame.DataFrame, sg: SchemaGenerator, restrict_rules: bool, project_scope: List, access_token: str = None,
```

Comment:
I don't know if you want to change it now or when we type annotate this, but `access_token: str = None` should be 
 `access_token: Optional[str] = None`

Reply:
thanks for catching, I'll fix this one now